skimage/io/tests/test_mpl_imshow.py

```diff
@@ -18,6 +18,12 @@ def setup():
 im_lo = imf / 1000
 im_hi = imf + 10
 
+the_expected_warnings = [r"tight_layout : falling back to Agg|\A\Z",
```

Comment:
Can you rename this to something more meaningful? e.g. `expected_imshow_warnings` or `expected_warnings_on_imshow`?

Reply:
Reading the later tests, I might suggests three constants:
- base expected warnings on imshow
- CObject warning on imshow
- low range warning on imshow

Then you can combine them with + in each test?

Reply:
this is alot of work, for a branch that isn't even master.

Reply:
Fair enough, hence why I didn't tick "request changes". =) The first suggestion is not a lot of work though! Actually let me do it with suggestions.

Reply:
```suggestion
expected_imshow_warnings = [r"tight_layout : falling back to Agg|\A\Z",
```

Reply:
Never mind, just saw that you took care of it! Thank you!! =)